scripts/travis_test.sh

```diff
@@ -5,7 +5,10 @@ RV=0
 # Google 2016 Unbreakable
 cd google2016_unbreakable
 time python win.py | tee unbreakable.log
-if grep -q "CTF{0The1Quick2Brown3Fox4Jumped5Over6The7Lazy8Fox9}" unbreakable.log
+RAN_OK=${PIPESTATUS[0]}
```

Comment:
what if we use `set -o pipefail` at the top? this would let us have less code because we wouldn't need to explicitly check the script's ret value each time.

https://unix.stackexchange.com/questions/14270/get-exit-status-of-process-thats-piped-to-another